### DISCUSSION TITLE
在使用splithttp时，偶尔会出现卡顿，在客户端弹出多个protocol error: received DATA after END_STREAM后恢复。

### DISCUSSION THREAD

**Discussion by zwzzrl** [XTLS/Xray-core#3586] [Q&A]
如题，在访问网站时的突发卡顿，服务器是splithttp+cdn（cf），似乎在打开新网页时更容易出现卡顿，开始以为是在服务端的Max Upload Size造成的，但是无论是调大调小都依然出现。客户端是v2rayN（xray v1.8.21）,服务器使用v1.8.21。客户端方面使用h2，服务器没有开启tls，由cf处理tls。

**Comment by RPRX** [comment]
其实我也有这样的体感，SplitHTTP 下行应该没问题，目前上行的问题比较大，待传输数据量大时应该会“卡顿” https://github.com/XTLS/Xray-core/issues/3560#issuecomment-2241816491

**Comment by kuraudowelle** [comment]
跟你有一模一样的问题，但是在最新的内核版本中基本上解决了，偶尔还是会出现不那么经常了

**Comment by mmmray** [comment]
@zwzzrl do you still encounter this issue? any other errors that happen recently?

**Reply by zwzzrl** [reply]
目前工作的非常好，已经没有再出现卡顿然后抛出大量DATA after END_STREAM，还没有发现其他问题。
